data_measurements/streamlit_utils.py

```diff
@@ -291,6 +291,26 @@ def expander_text_duplicates(dstats, column_id):
             st.dataframe(dstats.dup_counts_df.reset_index(drop=True))
 
 
+### Then, show perplexities
+def expander_text_perplexities(dstats, column_id):
+    # TODO: Saving/loading figure
+    with st.expander(f"Text Perplexities{column_id}", expanded=False):
+        st.caption(
+            "Use this widget to identify text perplexities from GPT-2."
+        )
+        st.markdown(
+            "Outlier perplexities could highlight an issue with an example."
```

Comment:
```suggestion
            """
            Outlier perplexities, especially very high values, could highlight an issue with an example.
            Smaller variations should be interpreted with more care, as they indicate how similar to
            the GPT-2 training corpus the examples are rather than being reflective of general linguistic properties.
            For more information the GPT-2, see its [model card](https://hf.co/gpt2).
            """
```